frontend/src/app/Catalog/CatalogFilters/CatalogFilters.module.scss

```diff
@@ -0,0 +1,195 @@
+.root {
+	display: flex;
+	flex-direction: column;
+	background: transparent;
+	overflow: hidden;
+
+	@include media(mobile, tablet) {
+		display: grid;
+		position: relative;
+		grid-template-columns: 1fr;
+		grid-template-rows: min-content 1fr;
+		grid-template-areas:
+			'toggle'
+			'modal';
+		padding-top: 0;
+		align-items: center;
+
+		&[data-modal='true'] {
+			overflow: visible !important;
+		}
+	}
+}
+
+.toggle {
+	display: none;
+
+	@include media(mobile, tablet) {
+		display: flex;
+		grid-area: toggle;
+		z-index: 200;
+		justify-content: space-between;
+		align-items: center;
+		justify-self: center;
+		transition-duration: 0.1s;
+		background: transparent;
+		border-radius: 0px 0px 12px 12px;
+		border: 1.5px solid #eaeaea;
+		border-top: none;
+		padding: 5px 8px;
+		background: white;
+		font-size: 14px;
+
+		input {
+			font-size: 14px;
+		}
+
+		button {
+			font-size: 18px;
+			outline: none;
+			color: #8a8a8a;
+			border: none;
+			padding: 5px;
+			background: white;
+			border-radius: 50%;
+			&:active,
+			&:hover {
+				color: $bt-blue;
+			}
+		}
+	}
+}
+
+.search {
+	@include media(mobile, tablet) {
+		display: none;
+	}
+}
+
+.container {
+	height: 100%;
+	overflow-y: scroll;
+
+	@include media(mobile, tablet) {
+		position: absolute;
+		grid-area: modal;
+		top: -48px;
+		width: 100%;
+		height: calc(100vh - 36px);
+		z-index: 50;
+		background: hsla(0, 0%, 0%, 0.3);
+		justify-content: center;
+		align-items: center;
+
+		&[data-modal='false'] {
+			display: none;
+		}
+
+		&[data-modal='true'] {
+			display: flex;
+		}
+	}
+}
+
+.wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: 12px;
+	background: white;
+	padding: 30px 20px 30px 30px;
+	border-right: 1.5px solid #eaeaea;
+	height: 100%;
+	font-size: 14px;
+	overflow-y: scroll;
+
+	@include media(mobile, tablet) {
+		position: absolute;
+		border-radius: 12px;
+		background: white;
+		width: 80%;
+		height: min-content;
+
+		&[data-modal='false'] {
+			display: none;
+		}
+
+		&[data-modal='true'] {
+			display: flex;
+		}
+	}
+}
+
+.item {
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
+	font-weight: 600;
+
+	p {
+		font-weight: 600;
+	}
+}
+
+.error {
+	background: $bt-indicator-red;
+	color: $bt-white;
+	font-size: 14px;
+	padding: 10px;
+	border-radius: 6px;
+	text-align: center;
+}
+
+.header {
+	margin-bottom: 20px;
+	display: flex;
+	justify-content: space-between;
+	flex-direction: row;
+	align-items: baseline;
+	gap: 30px;
+	h3 {
+		font-size: 20px;
+		line-height: 30px;
+		font-weight: 600;
+	}
+
+	button {
+		color: #2f80ed;
+		transition: 0.5s;
+		border: none;
+		background: none;
+		padding: 4px 8px;
+		border-radius: 4px;
+		font-size: 16px;
+		&:hover {
+			background: hsla(0, 0%, 92.2%, 0.6196078431);
+		}
+	}
+}
+
+.select {
```

Comment:
You changed the design for the select inputs by increasing the font-size and font-weight and changing the placeholder values, which are no longer capitalized. You should change this back to match the current design.